app/services/idv/steps/link_sent_step.rb

```diff
@@ -28,9 +28,9 @@ def handle_document_verification_failure(get_results_response)
       end
 
       def render_document_capture_cancelled
-        failure(I18n.t('errors.doc_auth.document_capture_cancelled'))
         mark_steps_incomplete
```

Comment:
Arguably we could remove `link_sent` from `HYBRID_FLOW_STEPS` since it's not having any effect here anyways, but I don't think it hurts anything to keep.

Reply:
thanks! I have no desire to touch this code any more than is absolutely needed, gonna leave as-is for now